src/ontology/mondo-ingest.Makefile

```diff
@@ -334,8 +334,15 @@ metadata/%-metrics.json: $(COMPONENTSDIR)/%.owl
 $(DOCS_DIR)/metrics/%.md: metadata/%-metrics.json | $(DOCS_DIR)/metrics/
 	jinjanate "$(SOURCE_METRICS_TEMPLATE)" metadata/$*-metrics.json > $@
 
+ALL_DBS=$(foreach n,$(ALL_COMPONENT_IDS), $(COMPONENTSDIR)/$(n).db)
+$(REPORTDIR)/iri-sources.txt $(REPORTDIR)/iri-sources.md: $(foreach n,$(ALL_COMPONENT_IDS), $(COMPONENTSDIR)/$(n).db)
```

Comment:
can you call the file `source-component-versions.txt` so it is more generally understandable what the file contains?

Reply:
I changed to that. However technically these are the versions of the mirrors/sources, not necessarily the component, so if you want me to rename it again, let me know.